deployment/grafana/provisioning.ts

```
@@ -0,0 +1,404 @@
+	 */
+	generateDockerCompose(): object {
+		return {
+			version: "3.8",
+			services: {
+				grafana: {
+					image: "grafana/grafana:latest",
+					container_name: "terragon-grafana",
+					restart: "unless-stopped",
+					ports: ["3001:3000"],
+					environment: {
+						GF_SECURITY_ADMIN_PASSWORD: process.env.GRAFANA_ADMIN_PASSWORD || "admin",
```

Biome (2.1.2)

[error] This object property name should be in camelCase.



(lint/style/useNamingConvention)